processor/coralogixprocessor/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package coralogixprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/coralogixprocessor"
+
+import "fmt"
+
+type sampingConfig struct {
```

Comment:
```suggestion
type samplingConfig struct {
```
Please update references as well

Reply:
[26e7c79](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33458/commits/26e7c791ce217ddffeca2e7b3f5e363dfebd96a8)